script/tap-driver

```diff
@@ -370,7 +370,7 @@ function setup_result_obj(line)
   sub("^(not )?ok[ \t]*", "", line)
 
   # If the result has an explicit number, get it and strip it; otherwise,
-  # automatically assing the next progresive number to it.
+  # automatically assing the next progressive number to it.
```

Comment:
This might need to be fixed somewhere else first, since it's a GPL'd script that is vendor'd in because of Automake.

Reply:
I'll just drop it from this patch :P